src/libraries/System.Threading/tests/SynchronizationContextTests.cs

```diff
@@ -37,14 +36,12 @@ public static void WaitTest()
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Mono)]
         public static void WaitTest_ChangedInDotNetCore()
         {
             Assert.Throws<ArgumentNullException>(() => TestSynchronizationContext.WaitHelper(null, false, 0));
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Mono, "https://bugzilla.xamarin.com/show_bug.cgi?id=60568")]
         public static void WaitNotificationTest()
```

Comment:
Looks like this test still fails. The bugzilla bug was marked as fixed so probably some other issue.

Can you please add a new issue for this and disable the test with `ActiveIssue`?

Reply:
Yeah, I'll do